src/ims/store/_db.py

```diff
@@ -674,10 +759,12 @@ async def incidents(self, eventID: str) -> Iterable[Incident]:
         """
 
         def incidents(txn: Transaction) -> Iterable[Incident]:
-            return tuple(
-                self._fetchIncident(eventID, number, txn)
-                for number in tuple(self._fetchIncidentNumbers(eventID, txn))
-            )
+            return self._fetchIncidents(eventID, txn)
+            # TODO: remove this if the _fetchIncidents way performs better
+            # return tuple(
+            #     self._fetchIncident(eventID, number, txn)
+            #     for number in tuple(self._fetchIncidentNumbers(eventID, txn))
+            # )
```

Comment:
Did you confirm this is better?

FWIW, if you add `--log-level=debug` to server startup and look for messages in the namespace `ims.store.mysql.*` (or `ims.store.sqlite.*`), you'll see all of the DB transactions are logged, eg:

```console
2024-10-31T11:19:49-0700 [ims.store.mysql._store.Cursor#debug] EXECUTE: 
	select VERSION from SCHEMA_INFO
	 <- {}
2024-10-31T11:19:49-0700 [ims.store._db.DatabaseManager#debug] No upgrade required for schema version 6
2024-10-31T11:19:49-0700 [ims.store.mysql._store.DataStore#info] Validating data store...
2024-10-31T11:19:50-0700 [ims.store.mysql._store.Cursor#debug] EXECUTE: 
	select AUTHOR, TEXT, CREATED, GENERATED from REPORT_ENTRY
	where
	    ID not in (select REPORT_ENTRY from INCIDENT__REPORT_ENTRY) and
	    ID not in (select REPORT_ENTRY from INCIDENT_REPORT__REPORT_ENTRY)
	 <- {}
```

Reply:
Yeah, I have been watching those logs locally, with a small bunch of incidents and incident entries
I can only run IMS locally and see the logs locally as things are, so my intention is to merge the PR to see the impact on API duration in staging